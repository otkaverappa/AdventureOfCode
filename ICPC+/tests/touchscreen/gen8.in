19
xebdowab 10
mtmubwxz
jshyfyts
wecgamjz
smcgkfkx
ayrcupdd
hnbmlvhh
zjpbyaaq
nfzxkjwk
hoocfsho
fjcsgjah
tritskjh 10
pkgbtdod
rchyuomb
zoufawmv
nwohhzow
kvydyoir
qrrnidoh
tiptebou
acbhcsdm
pdrptcik
tzxbellx
watadhwd 10
mykosqch
vvwpxebt
fywjljgh
jzkpjguv
ghlyzphw
kflklofq
obzalijw
jtnuaipi
rcjqrsmc
yaojqtcf
wdfjloix 10
kvrmfjuw
lfodydhw
dvhtqjym
nfwawezg
brshaofn
uwqszzqe
uxxkjyzy
fxabbaje
rdntstgo
rziqyyuv
xthitiib 10
fjegjolc
tyvltdbk
ejcfjwaj
sjtlscox
ltguhtwa
ttonwrzd
bcimakxu
vrintwne
ptaymwzf
ppungwsj
aaycmvxi 10
ofvhdknt
dprrlszc
jvsstkdt
nbyzzvhn
cfxfpmav
csmpmnuw
jmocyryl
vwnwtulx
bicrxdoz
xcqjqkhb
wvdxpdim 10
bviustrv
ewmbbccy
fsjvdrwb
obyfhhri
ezdyvwuz
siatlets
wdnzwlbk
ozqvihep
jjpeflfy
uhrhllzh
qojmzmyn 10
lolwypnh
ycndoubk
bvtoisva
ggmhskwg
bhezwriw
vxclrdxu
aqikifkq
oyzglwom
fsnelxaj
wcunisjk
ktvtahlo 10
gnxtllfq
guurrvan
zxchrnub
gpwiwkxe
xuxihean
avhtshir
gmbxzvbh
mxqkhnpg
joqqvrev
mlpeuxxd
kaclvdth 10
dlsmyjtj
zjzucfqo
shupeusq
uwcscxzf
itriemre
ysyaaqrs
xljehduc
ayucxvjf
paptpgzn
bynbogvo
shszkodm 10
pxomtxuk
ajfpsfet
ftwtatju
abtnszzh
wqvrnpdp
zjgrqkmv
eiqgbaae
bvrtuqcr
gykwnono
xufngrkm
basdasjc 10
oaxisabz
amxncmcb
gjqpdabe
aujbosfe
ucpneqog
enugzyhi
jxxmzyta
uebjxipt
megtvuzb
jvkjvtrf
totsoosj 10
tvusfjls
osllokoy
gyjdsail
qefesapl
vlddvrvl
jgwzskza
liedkoqc
uxjnxyav
kgyhxtsi
bqjvcjwn
tcsgskin 10
ircfsece
maljtftu
vfszqonl
shrkuczc
vbjngmrs
meeijzee
gwewmtig
cbrwdsaz
umocagwo
mcwwedak
zgjobrue 10
snawgdxc
plgptedg
hccnfezg
niuozrsu
evqmypon
auewzhci
lgxqmwxz
ftqekiar
fqddhssk
oygpikxt
sumetjgy 10
ewergfim
xnrfhkpw
kxlshlmb
hyfajnap
jfgqkqcj
etolffjq
exklkymt
wuuhhwxt
bfjnwnxc
inpovzgz
ysnirbbn 10
xxxfuwav
bjlzykbg
ztwuuews
xjaomddk
barvyrsa
cfzapdgp
ydlshhke
smuhpyrq
aknzbgbf
ncfdhluf
qhxarjgl 10
vbslbjdd
trewzfbm
kgrrunzm
vwmmgvxd
wsozbsex
jjtkqxwa
fqszdrma
paoxvmbt
gruhlzgw
icgyzfbe
vtgamubd 10
upbsdenj
xitjhzfr
borctshq
lnqzhrdd
hgxmmlwl
tpucqcts
qkulddbq
qrszlxdu
fagtneeg
tzkkdfev
