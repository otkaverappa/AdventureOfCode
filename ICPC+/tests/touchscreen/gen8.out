ayrcupdd 24
fjcsgjah 24
mtmubwxz 26
nfzxkjwk 26
smcgkfkx 26
hnbmlvhh 27
zjpbyaaq 27
wecgamjz 32
jshyfyts 33
hoocfsho 36
tiptebou 17
rchyuomb 19
kvydyoir 23
tzxbellx 25
qrrnidoh 26
zoufawmv 26
nwohhzow 28
acbhcsdm 29
pdrptcik 30
pkgbtdod 31
rcjqrsmc 23
yaojqtcf 23
fywjljgh 32
jzkpjguv 33
ghlyzphw 35
obzalijw 35
vvwpxebt 35
kflklofq 38
mykosqch 39
jtnuaipi 40
kvrmfjuw 22
brshaofn 27
rziqyyuv 27
dvhtqjym 28
fxabbaje 30
uxxkjyzy 31
rdntstgo 32
lfodydhw 35
nfwawezg 43
uwqszzqe 45
fjegjolc 22
sjtlscox 23
vrintwne 24
ltguhtwa 26
tyvltdbk 26
bcimakxu 31
ttonwrzd 32
ejcfjwaj 35
ppungwsj 38
ptaymwzf 38
nbyzzvhn 28
jmocyryl 29
cfxfpmav 30
csmpmnuw 31
dprrlszc 31
jvsstkdt 31
vwnwtulx 36
ofvhdknt 37
xcqjqkhb 38
bicrxdoz 46
siatlets 26
ezdyvwuz 27
wdnzwlbk 28
fsjvdrwb 30
obyfhhri 30
ozqvihep 30
ewmbbccy 31
uhrhllzh 34
bviustrv 37
jjpeflfy 42
ggmhskwg 23
wcunisjk 31
lolwypnh 33
aqikifkq 36
bhezwriw 37
ycndoubk 38
oyzglwom 41
vxclrdxu 41
fsnelxaj 42
bvtoisva 45
avhtshir 21
gmbxzvbh 26
mxqkhnpg 29
zxchrnub 30
guurrvan 31
gnxtllfq 32
gpwiwkxe 33
joqqvrev 35
xuxihean 36
mlpeuxxd 44
uwcscxzf 22
itriemre 24
dlsmyjtj 27
ysyaaqrs 29
zjzucfqo 29
bynbogvo 34
paptpgzn 34
ayucxvjf 35
xljehduc 36
shupeusq 37
xufngrkm 23
gykwnono 24
wqvrnpdp 24
ftwtatju 28
zjgrqkmv 29
ajfpsfet 32
bvrtuqcr 32
abtnszzh 33
eiqgbaae 33
pxomtxuk 48
oaxisabz 20
jxxmzyta 23
gjqpdabe 25
megtvuzb 30
jvkjvtrf 31
ucpneqog 31
enugzyhi 32
uebjxipt 32
amxncmcb 33
aujbosfe 34
liedkoqc 18
vlddvrvl 25
kgyhxtsi 29
bqjvcjwn 30
tvusfjls 30
jgwzskza 31
gyjdsail 34
osllokoy 36
uxjnxyav 36
qefesapl 42
gwewmtig 22
vfszqonl 22
vbjngmrs 26
ircfsece 27
mcwwedak 27
maljtftu 30
umocagwo 32
cbrwdsaz 33
meeijzee 35
shrkuczc 38
niuozrsu 24
plgptedg 29
snawgdxc 29
evqmypon 33
hccnfezg 33
auewzhci 34
ftqekiar 34
oygpikxt 34
fqddhssk 37
lgxqmwxz 38
wuuhhwxt 20
xnrfhkpw 24
ewergfim 25
hyfajnap 25
etolffjq 27
bfjnwnxc 28
exklkymt 28
kxlshlmb 28
jfgqkqcj 30
inpovzgz 38
ydlshhke 26
aknzbgbf 28
ncfdhluf 28
barvyrsa 29
bjlzykbg 29
xxxfuwav 31
xjaomddk 32
ztwuuews 35
cfzapdgp 36
smuhpyrq 36
trewzfbm 24
vbslbjdd 27
fqszdrma 29
kgrrunzm 30
paoxvmbt 34
wsozbsex 34
icgyzfbe 37
vwmmgvxd 38
gruhlzgw 41
jjtkqxwa 41
fagtneeg 22
lnqzhrdd 23
borctshq 25
upbsdenj 26
xitjhzfr 26
qrszlxdu 28
hgxmmlwl 29
tpucqcts 31
qkulddbq 33
tzkkdfev 34
